manifests/init.pp

```diff
@@ -94,7 +94,9 @@
   if $control_enable {
     Service[$service_name] {
       restart   => 'unbound-control reload',
+      require   => Class['unbound::remote'],
     }
+    Package[$package_name] -> Class['unbound::remote']
```

Comment:
Not all platforms have a package to install, so we'll need to provide some logic to work around that.